imap/jmap_contact.c

```diff
@@ -1782,6 +1753,18 @@ static json_t *jmap_contact_from_vcard(const char *userid,
         }
     }
 
+    if (defaultEmailIndex < 0)
```

Comment:
If defaultEmailIndex is reset to 0 here, why first initialize it to -1?

Reply:
The logic is directly from the original code, but changing the variable name from defaultIndex to defaultEmailIndex

Reply:
I think the logic is to only flag the first PREF that we see as the default, in case there are more than one.  If we know that if we initialize to zero and blindly set defaultIndex each time we see PREF, then we will get the last rather than the first.

I don't know if it makes any difference.

Reply:
Right! I did not think of that. So let's keep it initialized to -1.